src/js/components/LottoList.js

```diff
@@ -0,0 +1,24 @@
+import { proxyState } from '../store/index.js';
+import { registeReactiveRender } from '../utils/reactiveRender.js';
+import LottoItem from './LottoItem.js';
+
+export default function LottoList() {
+  const $lottoList = document.querySelector('#lotto-list');
+
+  const clearLottoList = () => {
+    while ($lottoList.firstChild) {
+      $lottoList.removeChild($lottoList.firstChild);
+    }
+  };
+
+  const render = () => {
+    clearLottoList();
+
+    proxyState.lottoList.forEach(lotto => {
+      $lottoList.appendChild(LottoItem({ lotto, visible: proxyState.isVisibleLottos }));
+    });
```

Comment:
```suggestion
    const $lottoListClone = $lottoList.cloneNode(true); // 메모리상에만 존재하는 DOM\
    proxyState.lottoList.forEach(lotto => {
      $lottoListClone.appendChild(LottoItem({ lotto, visible: proxyState.isVisibleLottos }));
    });
    $lottoList.replaceWith($lottoListClone); // DOM 교체
```

이런식으로도 작업할 수 있는데, 이 외에 또 어떤 방법이 있는지 고민해보시면 좋을 것 같아요!

Reply:
3가지의 방법으로 구현을 진행해보았습니다!
- replaceWith과 유사하지만 Children을 바꾸는 replaceChildren 이용하기 [dcb2191](https://github.com/next-step/js-lotto/pull/134/commits/dcb2191e6482e26d490f1c491584b819e5dd6b25)
- fragment 사용하여 list를 감싸도록한다음 한번만 append하기 [602cd57](https://github.com/next-step/js-lotto/pull/134/commits/602cd573e53c1da873c593f86d29b566ab0f1d31)
- template을 만들어 innerHTML로 한번만 랜더하기 [938bad4](https://github.com/next-step/js-lotto/pull/134/commits/938bad446ebd2f69d4f5ecdbc07a3fb3db209d2a)